lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -35,9 +35,10 @@ nv_overlay_evpn:
 
 vn_segment_vlan_based:
   # MT-lite only
-  /N(3|9)/:
+  N3k: &vn_segment_vlan_based_N3k
     kind: boolean
     config_get: 'show running section feature'
     config_get_token: '/^feature vn-segment-vlan-based$/'
     config_set: 'feature vn-segment-vlan-based'
     default_value: false
+  N9k: *vn_segment_vlan_based_N3k
```

Comment:
Got it. I was concerned that these would somehow drive changes to the NU APIs themselves but these anchors & aliases are only meaningful inside the yaml file itself. 
Consider adding a note or reference in README_YAML.md since this is bound to come up again (that's the first place I looked).